### PR TITLE
Further tuning of the Gentoo installation instructions.

### PR DESCRIPTION
**Pull request by paul-jewell**
Based on feedback from the Gentoo repository maintainer, addition of the possibility to install rust-analyzer via rustup.

### CODE REVIEW

File: docs/user/manual.adoc [hunk 1]
```diff
@@ -186,10 +186,12 @@ $ pacman -S rust-analyzer
...
 
-`rust-analyzer` is installed when the `rust-analyzer` USE flag is set for `dev-lang/rust` or `dev-lang/rust-bin`.
-You also need to set the `rust-src` USE flag.
+There are two ways to install `rust-analyzer` under gentoo:
 
-This is similar to using the `rustup` component, so it will install a version that lags behind the official releases on GitHub .
+- When installing dev-lang/rust, or dev-lang/rust-bin, set the `rust-analyzer` USE flag, as well as the `rust-src` USE flag.
+- Use the rustup component to install `rust-analyzer` (see instructions above).
+
+In both cases, the version installed lags behind the official releases on GitHub. If you need a newer version of `rust-analyzer`, then consider installing from source.
```
Comment (lnicola) [inline docs/user/manual.adoc:194]: We also have binaries on GitHub Releases.

Comment (paul-jewell) [reply docs/user/manual.adoc:194]: Thanks - I'll add that as well.

**Comment by paul-jewell** [comment]
Looks good - @lnicola - thanks for the improvements!

**Pull request merged**